docs/source/index.md

```diff
@@ -99,147 +137,56 @@ Explore our detailed CLV examples using data from the [`lifetimes`](https://gith
 
-We provide the following professional services:
+---
 
-- **Custom Models**: We tailor niche marketing anayltics models to fit your organization's unique needs.
-- **Build Within PyMC-Marketing**: Our team are experts leveraging the capabilities of PyMC-Marketing to create robust marketing models for precise insights.
-- **SLA & Coaching**: Get guaranteed support levels and personalized coaching to ensure your team is well-equipped and confident in using our tools and approaches.
-- **SaaS Solutions**: Harness the power of our state-of-the-art software solutions to streamline your data-driven marketing initiatives.
+### More [PyMC Labs](https://www.pymc-labs.com/) Blog Posts and Resources
 
-## Support
+#### Marketing Mix Models
 
-This repository is supported by [PyMC Labs](https://www.pymc-labs.io).
+- [Unobserved Confounders, ROAS and Lift Tests in Media Mix Models](https://www.pymc-labs.com/blog-posts/mmm_roas_lift/)
 
-For companies that want to use PyMC-Marketing in production, [PyMC Labs](https://www.pymc-labs.io) is available for consulting and training. We can help you build and deploy your models in production. We have experience with cutting edge Bayesian modelling techniques which we have applied to a range of business domains including marketing analytics.
+#### Customer Lifetime Value
 
-:::{image} _static/labs-logo-dark.png
-:align: center
-:target: https://www.pymc-labs.io
-:scale: 20 %
-:alt: PyMC Labs logo
-:class: only-dark
-:::
+- [Hierarchical Customer Lifetime Value Models](https://www.pymc-labs.com/blog-posts/hierarchical_clv/)
+- [Customer Lifetime Value in the non-contractual continuous case: The Bayesian Pareto NBD Model](https://www.pymc-labs.com/blog-posts/pareto-nbd/)
+- [Cohort Revenue & Retention Analysis](https://www.pymc-labs.com/blog-posts/cohort-revenue-retention/)
 
-:::{image} _static/labs-logo-light.png
-:align: center
-:target: https://www.pymc-labs.io
-:scale: 20 %
-:alt: PyMC Labs logo
-:class: only-light
-:::
+### Case Studies
 
+- [Building an in-house marketing analytics solution](https://www.pymc-labs.com/blog-posts/2023-07-18-niall-In-house-marketing/)
+- [Bayesian Media Mix Models: Modelling changes in marketing effectiveness over time](https://www.pymc-labs.com/blog-posts/modelling-changes-marketing-effectiveness-over-time/)
+- [Improving the Speed and Accuracy of Bayesian Media Mix Models](https://www.pymc-labs.com/blog-posts/reducing-customer-acquisition-costs-how-we-helped-optimizing-hellofreshs-marketing-budget/)
+- [Bayesian Media Mix Modeling for Marketing Optimization](https://www.pymc-labs.com/blog-posts/bayesian-media-mix-modeling-for-marketing-optimization/)
+- [Bayesian inference at scale: Running A/B tests with millions of observations](https://www.pymc-labs.com/blog-posts/bayesian-inference-at-scale-running-ab-tests-with-millions-of-observations/)
+For more blogposts and resources, check out the [PyMC Labs Blog](https://www.pymc-labs.com/blog-posts/).
```

Comment:
Just need an extra blank line so that this last line appears on a new line.

Reply:
Good catch! Fixed in https://github.com/pymc-labs/pymc-marketing/pull/1023/commits/f8f1af230f0e22294200bbc32c6fcda5ab440e7e